cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -66,7 +75,7 @@
   </my-timeline>
 
   <!-- LOGS TABLE -->
-  <div class="logs-table">
+  <div class="logs-table" ng-class="{'navigator-fullscreen-log-table' : LogViewer.entityName === 'Cloudera Navigator' }">
```

Comment:
I don't think this should have a specific class at the directive level. Can you target this from the container template?